metrics-operator/converter/slo_converter_test.go

```diff
@@ -565,52 +1150,72 @@ func TestSetupTarget(t *testing.T) {
 			wantErr: false,
 		},
 		{
-			name: "criteria with % -> informative slo",
+			name: "no warn criteria single - conversion",
 			o: &Objective{
 				Name: "criteria",
 				Pass: []Criteria{
 					{
-						Operators: []string{"<10%"},
+						Operators: []string{"<10"},
```

Comment:
I think here we don't test for bogus operator, e.g. `<<<10`
```

Reply:
good catch!